File: frontend/src/app/models/weekly-summary.ts
import { StandupEntry } from './standup-entry';

export interface WeeklySummary {
  weekStart: string;
  weekEnd: string;
  standupCount: number;
  blockersRaised: number;
  blockersResolved: number;
  resolutionRate: number;
  entries: StandupEntry[];
}

File: backend/StandupTracker.Api/Models/WeeklySummary.cs
namespace StandupTracker.Api.Models;

public record WeeklySummary(
    DateTime WeekStart,
    DateTime WeekEnd,
    int StandupCount,
    int BlockersRaised,
    int BlockersResolved,
    double ResolutionRate,
    IReadOnlyList<StandupEntry> Entries);
